types/next-pwa/index.d.ts

```diff
@@ -129,7 +129,7 @@ interface PWAConfig extends WebpackConfigOptions {
     /**
      * Array of Glob patterns to exclude files inside `public` _(or custom `dest`)_  folder from being precached.
      *
-     * @default ["!noprecache/**‎/*"]
+     * @default ["!noprecache/** /*"]
```

Comment:
[Still broken](https://dev.azure.com/definitelytyped/DefinitelyTyped/_build/results?buildId=145119&view=logs&j=8635df7d-ecde-52a7-c7a8-84b998e35690&t=c264d665-0b79-59b7-7e04-9965cbe74860&l=92). How should I tackle this?

Reply:
maybe you can try this one
```
@default ["/!noprecache/**\/*"]
```

Reply:
Tried it, but the `\` is picked up by the parser, probably because it is read as a string instead of a regex.

I chose to [remove the default value description](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/63391/commits/4630df86b2a656024fcc0d51ce9162ea4fc8aacf) and instead include a reference to the original package's docs about the property.

Reply:
I read through a few issues, mainly [tsdoc issue#166](https://github.com/microsoft/tsdoc/issues/166) and [jsdoc escape char mechanism](https://github.com/gajus/eslint-plugin-jsdoc/issues/710), and it seems that there is no solution, as of yet. It May be a change in the near future, so I'll be sure to keep an eye on it.